build/build-extensions.ps1

```diff
@@ -155,6 +158,16 @@ function CreateSiteExtensions() {
     Remove-Item $siteExtensionPath -Recurse -Force > $null
 }
 
+function WriteHashesFile([string] $directoryPath) {
+  New-Item -Path "$directoryPath\..\temp_hashes" -ItemType Directory
+  $temp_current = (Get-Location)
+  Set-Location $directoryPath
+  Get-ChildItem -Recurse $directoryPath | where { $_.PsIsContainer -eq $false } | Foreach-Object { Write-Output ("Hash:" + [System.Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes((Get-FileHash -Algorithm MD5 $_.FullName).Hash)) + " FileName:" + (Resolve-Path -Relative -Path $_.FullName)) } | Out-File -FilePath "$directoryPath\..\temp_hashes\hashes.txt"
```

Comment:
You can omit Write-Output. This would do the same:
`"Hash:" + [System.Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes((Get-FileHash -Algorithm MD5 $_.FullName).Hash)) + " FileName:" + (Resolve-Path -Relative -Path $_.FullName)`